bench/wtperf/wtperf.c

```diff
@@ -387,21 +405,29 @@ worker(void *arg)
 	WT_SESSION *session;
 	int64_t ops, ops_per_txn, throttle_ops;
 	size_t i;
-	uint64_t next_val, usecs;
+	uint64_t final_milestone_gap, needed_milestones, next_val;
```

Comment:
Could you split the truncate part out into a different function - it makes the ```worker``` function ever more complex.